app/src/main/java/com/cornellappdev/uplift/di/AppModule.kt

```diff
@@ -0,0 +1,32 @@
+package com.cornellappdev.uplift.di
+
+import com.apollographql.apollo3.ApolloClient
+import com.cornellappdev.uplift.BuildConfig
+import com.cornellappdev.uplift.data.clients.ApolloReportClient
+import com.cornellappdev.uplift.domain.clients.ReportClient
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+
+@Module
+@InstallIn(SingletonComponent::class)
+object AppModule {
+
+    @Provides
+    @Singleton
+    fun provideApolloClient(): ApolloClient {
+        return ApolloClient.Builder()
+            .serverUrl(BuildConfig.BACKEND_URL)
+            .build()
+    }
+
```

Comment:
Love it, using this syntax to provide a streamlined apollo client. Super clean stuff.